src/styles/stars.scss

```
@@ -0,0 +1,27 @@
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+  :last-child {
+    margin-right: 0;
+  }
+
+  :hover {
+    background-image: url(../images/star-active.svg);
+  }
```

specify what exactly you add these pseudo's
```suggestion
  &:last-child {
    margin-right: 0;
  }

  &:hover {
    background-image: url(../images/star-active.svg);
  }
```